Exercise_A/index.html

```diff
@@ -3,13 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Tony's Alter Ego</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
   </head>
   <body>
     <!-- CODE HERE -->
+    <figure>
+      <a href="about.html"><img src="dalmatian_in_green.jpg" alt="Tony Lo" width="200px" height="300px"/></a>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!